### ISSUE TITLE
Cloudflare protection

### ISSUE THREAD

**Issue by gfaessler** [Overmiind/Puppeteer-sharp-extra#16]
Thank you very much for the port. I am struggling to bypass cloudflare when using headless mode. When non headless it passes cloudflare but when headless it remains stuck on the cloudflare redirection page.

I prepared a short exemple below to reproduce the issue encountered. I am not sure if this works on the origin Node JS plugin.

```csharp
// Initialization plugin builder
var extra = new PuppeteerExtra();

// Use stealth plugin
extra.Use(new StealthPlugin());

// Launch the puppeteer browser with plugins
var versionCloudFlare = new BrowserFetcher().DownloadAsync(BrowserFetcher.DefaultRevision).Result;

// Headless test
var browser = await extra.LaunchAsync(new LaunchOptions()
{
    Headless = true,
    ExecutablePath = versionCloudFlare.ExecutablePath               
});

var page = await browser.NewPageAsync();
await page.GoToAsync("http://www.overclockers.co.uk");
await page.WaitForTimeoutAsync(10000);
await page.ScreenshotAsync("cloudflare-headless.png");

// Non Headless test
browser = await extra.LaunchAsync(new LaunchOptions()
{
    Headless = false,
    ExecutablePath = versionCloudFlare.ExecutablePath
});

page = await browser.NewPageAsync();
await page.GoToAsync("http://www.overclockers.co.uk");
await page.WaitForTimeoutAsync(10000);
await page.ScreenshotAsync("cloudflare-non-headless.png");
```
![cloudflare-headless](https://user-images.githubusercontent.com/75208847/110777896-f2b02200-8261-11eb-8eac-d72668ea267d.png)
![cloudflare-non-headless](https://user-images.githubusercontent.com/75208847/110777898-f348b880-8261-11eb-8e4c-af051c44383e.png)

**Comment by Overmiind** [comment]
Hello, I got the same result, but I have no idea why is this happening in headless mode. I would appreciate it If you give me any ideas to solve it

**Comment by gfaessler** [comment]
Thanks. I have to do some tests with the the Node JS plugin and dig further

**Comment by Overmiind** [comment]
Fixed by #20. Please update your NuGet package to version 1.3.0
